src/main/java/com/codecool/keepcash/ExternalApis/Controller/ExchangeRatesController.java

```
@@ -0,0 +1,28 @@
+package com.codecool.keepcash.ExternalApis.Controller;
+
+import com.codecool.keepcash.ExternalApis.Client.ExchangeRatesClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+@RestController
+public class ExchangeRatesController {
+
+    private ExchangeRatesClient exchangeRatesClient;
+
+    public ExchangeRatesController(ExchangeRatesClient exchangeRatesClient) {
+        this.exchangeRatesClient = exchangeRatesClient;
+    }
+
+    @GetMapping("/exchangeRates")
+    public String getRates() throws InterruptedException, IOException, URISyntaxException {
```

these exceptions are not handled, it will return stacktrace to the client